Sources/Foundation/FileHandle.swift

```diff
@@ -49,10 +49,10 @@ extension NSError {
  
 open class FileHandle : NSObject {
 #if os(Windows)
-    private var _handle: HANDLE
+    internal private(set) var handle: HANDLE
 
```

Comment:
Cant this be done as `public private(set) var _handle: HANDLE` and then the property getter isnt needed or am I missing something?

Reply:
It can, but would be a much larger change as there are other files that use `handle`.  I don't remember why there was a `_handle` and `handle` accessor.  I just made `handle` the internal interface since its used outside of this file and `_handle` the accessor since that needs to be underscored for the public exposure.

Reply:
If there is an alias for the same variable would it be worth the effort to fix this to avoid future confusion?

Reply:
I can live with that, updated the diffs.